### PR TITLE
Renamed CustomerID to clientID in baggage.md

### PR DESCRIPTION
**Pull request by HsantoshaKumara**

### CODE REVIEW

**Review by theletterf — APPROVED** [review]

**Comment by theletterf** [comment]
@open-telemetry/specs-semconv-approvers Please have a look.

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
Hmm, for this one I don't think we need SIG approval, as it's just updating the example text to be consistent with the image.

**Pull request merged**
